airbyte-integrations/connectors/source-hubspot/README.md

```diff
@@ -1,51 +1,88 @@
 # HubSpot Source
 
-This is the repository for the HubSpot source connector, written in Python.
-For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.io/integrations/sources/hubspot).
+This is the repository for the HubSpot source connector, written in Python. For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.io/integrations/sources/hubspot).
+
+## Primary keys
+
+The primary key for the following streams is `id`:
+
+- campaigns
+- companies
+- contacts
+- deals
+- email_events
+- engaments
+- engagements_calls
+- engagements_emails
+- engagements_meetings
+- engagements_notes
+- engagements_tasks
+- feedback_submissions
+- forms
+- line_items
+- marketing_emails
+- owners
+- products
+- tickets
+- ticket_pipelines
+- workflows
+- quotes
+
+The following streams do not have a primary key:
```

Comment:
could you add something about _why_ they don't have a primary key, potentially linking to any API docs or github issue that help the next person find out what's required to solve the issue?